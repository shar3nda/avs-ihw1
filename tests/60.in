15
6 -2 -2 8 2 3 0 -2 -2 0 5 6 4 6 4