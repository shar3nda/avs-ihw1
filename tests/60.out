-5 0 9 0 -3 -4 -5 10 1 -6